### PR TITLE
fix: add missing Lang method

### PR DESCRIPTION
**Pull request by devhaozi**
## 📑 Description

Closes https://github.com/goravel/goravel/issues/?

## Summary by CodeRabbit

- **New Features**
  - Introduced a new `Lang` function to handle language-specific paths.

- **Tests**
  - Added `TestLang` to ensure the `Lang` function operates correctly.

## ✅ Checks

- [x] Added test cases for my code

### CODE REVIEW

**Comment by devhaozi** [comment]
@coderabbitai full review

File: cache/memory_test.go [hunk 1]
```diff
@@ -67,23 +67,23 @@ func (s *MemoryTestSuite) TestDecrement() {
 }
 
 func (s *MemoryTestSuite) TestDecrementWithConcurrent() {
-	res, err := s.memory.Decrement("decrement")
+	res, err := s.memory.Decrement("decrement_concurrent")
```
Comment (hwbrzzl) [inline cache/memory_test.go:70]: Is there any difference between `decrement` and `decrement_concurrent`?

Comment (devhaozi) [reply cache/memory_test.go:70]: > Is there any difference between `decrement` and `decrement_concurrent`?

No more, but in the redis test case, the data is not cleared, so the test results are inconsistent.

File: go.mod [hunk 2]
```diff
@@ -1,10 +1,10 @@
 module github.com/goravel/framework
 
-go 1.22
+go 1.21
 
 require (
 	github.com/RichardKnop/machinery/v2 v2.0.13
-	github.com/brianvoe/gofakeit/v7 v7.0.4
+	github.com/brianvoe/gofakeit/v6 v6.28.0
```
Comment (hwbrzzl) [inline go.mod:7]: Can we use v7, but keep go 1.21?

Comment (devhaozi) [reply go.mod:7]: > Can we use v7, but keep go 1.21?

Can't, go mod tidy will auto update it.

**Review by hwbrzzl — APPROVED** [review]
Great 👍

**Pull request merged**